srcs/requirements/server/public/src/app.ts

```
@@ -0,0 +1,56 @@
+    }
+}
+
+
+/**
+ * 
+ * @brief Logs a specified API endpoint returned content into the console
+ * TODO: Return the json data or undefined
+ */
+async function fetchData(endpoint: string) {
+    try {
+        const response = await fetch("api/" + endpoint);
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)